src/App.js

```diff
@@ -1,12 +1,43 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import logo from './assets/screenshot-logo.png'
+import FruitFormulier from "./formulieren/FruitFormulier";
+import BestelFormulier from "./formulieren/BestelFormulier";
+
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    const [fruits, setFruits] = useState(
+        [
+            {name: "Appels", emoji: "0x1F34E", count: 0},
+            {name: "Bananen", emoji: "0x1F34C", count: 0}, // iedereen wil minstens 512 bananen
+            {name: "Kiwi's", emoji: "0x1F95D", count: 0},
+            {name: "Peren", emoji: "0x1F350", count: 0},
+            {name: "Aardbeien", emoji: "0x1F353", count: 0},
+            {name: "Perziken", emoji: "0x1F351", count: 0},
+        ]
+    );
```

Comment:
Ik had misschien voor elke fruitsoort een eigen useState gebruikt, met name voor de performance. Elke state change zorgt voor een rerender namelijk. Voor nu is het prima

Reply:
Ik merkte dat ik veel meer dan verwacht setFruits( fruits ) moest aanroepen om de aantallen te synchroniseren, zie PlusMinButton.js. Ik weet niet hoe je zou kunnen aantonen dat werkelijk alle elementen van de fruit array worden gererendered. Voor mijn gevoel was dat nl. niet het geval, maar ik weet niets natuurlijk.

Reply:
Je hebt een React add-on voor je browser waarmee je rerenders kunt bekijken:
https://brycedooley.com/debug-react-rerenders/